src/main/kotlin/io/github/gradlenexus/publishplugin/DefaultNexusRepositoryContainer.kt

```diff
@@ -39,10 +40,28 @@ internal open class DefaultNexusRepositoryContainer @Inject constructor(
     }
 
     override fun configure(configureClosure: Closure<*>): NamedDomainObjectContainer<NexusRepository> =
-        @Suppress("DEPRECATION") // TODO https://github.com/gradle-nexus/publish-plugin/issues/152
-        org.gradle.util.ConfigureUtil.configureSelf(
-            configureClosure,
-            this,
-            NamedDomainObjectContainerConfigureDelegate(configureClosure, this)
-        )
+        if (GradleVersion.current() < GradleVersion.version("7.6")) {
```

Comment:
@vlsi do you think we need this?
```suggestion
        if (GradleVersion.current().baseVersion < GradleVersion.version("7.6")) {
```

Reply:
Gave it a try:
```
println(GradleVersion.current()) // Gradle 8.0-rc-5
println(GradleVersion.current() < GradleVersion.version("8.0")) // true
println(GradleVersion.current() <= GradleVersion.version("8.0")) // true
println(GradleVersion.current().baseVersion < GradleVersion.version("8.0")) // false
println(GradleVersion.current().baseVersion <= GradleVersion.version("8.0")) // true
```

So based on that false, we need `.baseVersion`, because `7.6-...` versions should be considered "7.6".